packages/vips/package.json

```
@@ -0,0 +1,40 @@
+{
+	"name": "@wordpress/vips",
+	"version": "1.0.0-prerelease",
+	"private": true,
+	"description": "Utils for working with libvips.",
+	"author": "The WordPress Contributors",
+	"license": "GPL-2.0-or-later",
+	"keywords": [
+		"wordpress",
+		"media",
+		"libvips"
+	],
+	"homepage": "https://github.com/WordPress/gutenberg/tree/HEAD/packages/vips/README.md",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/WordPress/gutenberg.git",
+		"directory": "packages/vips"
+	},
+	"bugs": {
+		"url": "https://github.com/WordPress/gutenberg/issues"
+	},
+	"engines": {
+		"node": ">=18.12.0",
+		"npm": ">=8.19.2"
+	},
+	"main": "build/index.js",
+	"module": "build-module/index.js",
+	"types": "build-types",
+	"exports": {
+		".": {
+			"default": "./src/index.ts"
+		}
+	},
```

This seems new, I don't think any other packages use it. That's not a bad thing and we're considering using exports for building more script modules. CC @gziolo 👀 

Since this is unused so far in WordPress packages, I'd prefer not to introduce it here. It can be added to all packages at the same time in the same way when we decide to do that.

---

I am surprised to see it point to a TypeScript file. I think this should point to a compiled file, probably the same as the `module` field:

```suggestion
	"exports": {
		".": {
			"default": "./build-module/index.js"
		}
	},
```

When the field is defined like this with only a single `.` export, there's a shorthand form: `"exports": "./build-module/index.js"`.

---

This is mostly leftover from testing I think, agree it should be removed for consistency :-)

But my main focus right now is getting the WASM thing to work 😄